demo/ts/components/victory-zoom-container-demo.tsx

```diff
@@ -187,8 +176,10 @@ export default class VictoryZoomContainerDemo extends React.Component<
   }
 
   getZoomDomain() {
+    const yZoomDomain: RangeTuple = [random(0, 0.4), random(0.6, 1)];
```

Comment:
This wants declaring as a 2 item list rather than a list of indeterminate length with numbers... sure, typescript

Reply:
oh wow. 🎉